diff --git a/src/SamlService.php b/src/SamlService.php
index b068e3d..5de1603 100644
--- a/src/SamlService.php
+++ b/src/SamlService.php
@@ -371,6 +371,7 @@ class SamlService {
         'authnRequestsSigned' => (bool) $config->get('security_authn_requests_sign'),
         'wantMessagesSigned' => (bool) $config->get('security_messages_sign'),
         'requestedAuthnContext' => (bool) $config->get('security_request_authn_context'),
+        'wantNameId' => (bool) $config->get('wantNameId'),
       ),
       'strict' => (bool) $config->get('strict'),
     );
