diff --git a/src/Deskpro/API/APIResponse.php b/src/Deskpro/API/APIResponse.php
--- a/src/Deskpro/API/APIResponse.php
+++ b/src/Deskpro/API/APIResponse.php

@@ -105,7 +105,7 @@
      * @param mixed $offset
      * @param mixed $value
      */
-    public function offsetSet($offset, $value)
+    public function offsetSet(mixed $offset, mixed $value): void
     {
         if (is_null($offset)) {
             $this->data[] = $value;
@@ -120,7 +120,7 @@
      * @param mixed $offset
      * @return bool
      */
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->data[$offset]);
     }
@@ -130,7 +130,7 @@
      * 
      * @param mixed $offset
      */
-    public function offsetUnset($offset)
+    public function offsetUnset(mixed $offset): void
     {
         unset($this->data[$offset]);
     }
@@ -141,7 +141,7 @@
      * @param mixed $offset
      * @return null
      */
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
     {
         return isset($this->data[$offset]) ? $this->data[$offset] : null;
     }
@@ -149,7 +149,7 @@
     /**
      * Implements Iterator::rewind
      */
-    public function rewind()
+    public function rewind(): void
     {
         $this->it = 0;
     }
@@ -159,7 +159,7 @@
      * 
      * @return mixed
      */
-    public function current()
+    public function current(): mixed
     {
         return $this->data[$this->it];
     }
@@ -169,7 +169,7 @@
      * 
      * @return int
      */
-    public function key()
+    public function key(): int
     {
         return $this->it;
     }
@@ -177,7 +177,7 @@
     /**
      * Implements Iterator::next
      */
-    public function next()
+    public function next(): void
     {
         ++$this->it;
     }
@@ -187,7 +187,7 @@
      * 
      * @return bool
      */
-    public function valid()
+    public function valid(): bool
     {
         return isset($this->data[$this->it]);
     }
@@ -197,7 +197,7 @@
      * 
      * @return int
      */
-    public function count()
+    public function count(): int
     {
         return $this->count;
     }
