diff --git a/web/themes/custom/hoeringsportal/templates/layout/html.html.twig b/web/themes/custom/hoeringsportal/templates/layout/html.html.twig
index c92d050..ac7cd13 100755
--- a/web/themes/custom/hoeringsportal/templates/layout/html.html.twig
+++ b/web/themes/custom/hoeringsportal/templates/layout/html.html.twig
@@ -28,7 +28,6 @@
   <head>
     <head-placeholder token="{{ placeholder_token|raw }}">
     <title>{{ head_title|safe_join(' | ') }}</title>
-    <script id="CookieConsent" src="https://policy.app.cookieinformation.com/uc.js" data-culture="DA" type="text/javascript"></script>
     <css-placeholder token="{{ placeholder_token|raw }}">
     <js-placeholder token="{{ placeholder_token|raw }}">
     <script defer src="https://use.fontawesome.com/releases/v5.0.10/js/solid.js" integrity="sha384-Q7KAHqDd5trmfsv85beYZBsUmw0lsreFBQZfsEhzUtUn5HhpjVzwY0Aq4z8DY9sA" crossorigin="anonymous"></script>
