diff --git a/web/modules/custom/hoeringsportal_deskpro/hoeringsportal_deskpro.module b/web/modules/custom/hoeringsportal_deskpro/hoeringsportal_deskpro.module
index d5026c34..5ce33a38 100644
--- a/web/modules/custom/hoeringsportal_deskpro/hoeringsportal_deskpro.module
+++ b/web/modules/custom/hoeringsportal_deskpro/hoeringsportal_deskpro.module
@@ -73,6 +73,7 @@ function hoeringsportal_deskpro_theme() {
  * Implements hook_form_alter().
  */
 function hoeringsportal_deskpro_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  return;
   if ($form_id === 'node_hearing_form' || $form_id === 'node_hearing_edit_form') {
     // Department as select control.
     if (isset($form['field_deskpro_department_id']['widget'][0]['value']['#type'])
diff --git a/web/modules/custom/hoeringsportal_deskpro/src/Plugin/Validation/Constraint/AgentEmailContraintValidator.php b/web/modules/custom/hoeringsportal_deskpro/src/Plugin/Validation/Constraint/AgentEmailContraintValidator.php
index a80a11fd..1c8e335b 100644
--- a/web/modules/custom/hoeringsportal_deskpro/src/Plugin/Validation/Constraint/AgentEmailContraintValidator.php
+++ b/web/modules/custom/hoeringsportal_deskpro/src/Plugin/Validation/Constraint/AgentEmailContraintValidator.php
@@ -41,6 +41,7 @@ public static function create(ContainerInterface $container) {
    * {@inheritDoc}
    */
   public function validate($value, Constraint $constraint) {
+    return;
     // This is a single-item field so we only need to
     // validate the first item.
     $item = $value->first();
diff --git a/web/modules/custom/hoeringsportal_deskpro/src/Service/DeskproService.php b/web/modules/custom/hoeringsportal_deskpro/src/Service/DeskproService.php
index e020e97c..cd206dfb 100644
--- a/web/modules/custom/hoeringsportal_deskpro/src/Service/DeskproService.php
+++ b/web/modules/custom/hoeringsportal_deskpro/src/Service/DeskproService.php
@@ -531,6 +531,8 @@ public function getDefaultTicketLanguage() {
    * Convenience function for getting data from the Deskpro client.
    */
   private function get($endpoint, array $query = []) {
+    return new ApiResponse([], [], []);
+
     $cache = \Drupal::cache('data');
     $cacheKey = __METHOD__ . '||' . json_encode(func_get_args());
     $cacheTtl = $this->config->getCacheTtl();
